app/utils/validation_exception.js

```diff
@@ -0,0 +1,9 @@
+class ValidationException extends Error {
```

Comment:
if you want to use different types of exceptions like in Java, then do something similar https://docs.nestjs.com/exception-filters
or you can return `new Exception` everywhere with different messages and status codes